src/utilities/__tests__/mergeAST-test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ *  Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ *
+ */
+
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+import { parse, print } from '../../index';
+
+import { mergeAST } from '../mergeAST';
+
+import { fixtures } from './mergeAST-fixture';
+
+describe('MergeAST', () => {
+  fixtures.forEach(fixture => {
+    it(fixture.desc, () => {
+      const result = print(mergeAST(parse(fixture.query))).replace(/\s/g, '');
+      expect(result).to.equal(fixture.mergedQuery.replace(/\s/g, ''));
+    });
```

Comment:
We don't use fixtures please unroll it here.
Use `dedent` to remove indentation from strings.
And defined `expectMerged` wrapper to not repeat `print(mergeAST(parse`

Reply:
Please write it like this:
```js
function expectInlined(query) {
  return expect(print(inlineNamedFragments(parse(query))));
}

describe('inlineNamedFragments', () => {
  it('does not modify query with no fragments', () => {
    expectInlined(dedent`
      {
        id
      }
    `).to.equal(dedent`
      {
        id
      }
    `);
  });
```